java/code/src/com/redhat/rhn/common/conf/ConfigDefaults.java

```diff
@@ -209,6 +209,12 @@ public class ConfigDefaults {
      */
     private static final String TASKOMATIC_CHANNEL_REPODATA_WORKERS = "java.taskomatic_channel_repodata_workers";
 
+    /**
+     * TLS protocols which should be allowed by Uyuni server
+     */
+
+    private static final String TLS_PROTOCOLS_SUPPORTED_BY_SERVER = "java.supported.tls.protocols";
```

Comment:
dots are separators. I would suggest to use underscore as for the other values:

`java.supported_tls_protocols`